social-web-maintenance-wg-charter.html

```diff
@@ -0,0 +1,563 @@
+          <a href="https://www.w3.org/TR/activitystreams-vocabulary/">Activity Vocabulary</a>,
+          <a href="https://www.w3.org/TR/micropub/">MicroPub</a>,
+          <a href="https://www.w3.org/TR/ldn/">Linked Data Notifications</a>, and
+          <a href="https://www.w3.org/TR/webmention/">Webmention</a>
+          specifications, as well as related Notes.
+
+        <section id="section-out-of-scope">
+          <h3 id="out-of-scope">Out of Scope</h3>
+          <p>
+            Changes that add new functionality (<a href="https://www.w3.org/policies/process/#class-4">class 4</a>) are out of scope.
+          </p>
+        </section>
+
+      </section>
+
+      <section id="deliverables">
+        <h2>
+          Deliverables
+        </h2>
+
+        <p>Updated document status is available on the
+            <a href="https://www.w3.org/groups/wg/social/publications/">group publication status page</a>.
+        </p>
+
+        <section id="normative">
+          <h3>
+            Normative Specifications
+          </h3>
+          <p>
+            The Working Group will deliver the following W3C normative specifications:
+          </p>
+          <dl>
+            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/activitypub/">ActivityPub</a></dt>
+            <dd>
+              <p><b>Adopted Draft:</b> @@to be filled later@@
+              <p><b>Exclusion Draft:</b>  @@to be filled later@@
+                Associated @@to be filled later@@
+              <p><b>Exclusion Draft Charter:</b>  @@to be filled later@@
+
+            </dd>
+            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/websub/">WebSub</a></dt>
+            <dd>
+              <p><b>Adopted Draft:</b> @@to be filled later@@
+              <p><b>Exclusion Draft:</b>  @@to be filled later@@
+                Associated @@to be filled later@@
+              <p><b>Exclusion Draft Charter:</b>  @@to be filled later@@
+            </dd>
+            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/activitystreams-core/">Activity Streams</a></dt>
+            <dd>
+              <p><b>Adopted Draft:</b> @@to be filled later@@
+              <p><b>Exclusion Draft:</b>  @@to be filled later@@
+                Associated @@to be filled later@@
+              <p><b>Exclusion Draft Charter:</b>  @@to be filled later@@
+            </dd>
+            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/activitystreams-vocabulary/">Activity Vocabulary</a></dt>
+            <dd>
+              <p><b>Adopted Draft:</b> @@to be filled later@@
+              <p><b>Exclusion Draft:</b>  @@to be filled later@@
+                Associated @@to be filled later@@
+              <p><b>Exclusion Draft Charter:</b>  @@to be filled later@@
+
+            </dd>
+            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/micropub/">MicroPub</a></dt>
```

Comment:
Maybe this is nitpicky but we should use the lowercase-p Micropub capitalization everywhere so it's consistent with the spec.

Reply:
```suggestion
            <dt id="activity" class="spec"><a href="https://www.w3.org/TR/micropub/">Micropub</a></dt>
```

Reply:
Good call, thanks @manton. Fixed it.